src/session.rs

```diff
@@ -194,6 +195,35 @@ impl Session {
             .await
     }
 
+    /// Connect to the host at the given `host` over SSH using process impl, which will
+    /// spawn a new ssh process for each `Child` created.
+    ///
+    /// The format of `destination` is the same as the `destination` argument to `ssh`. It may be
+    /// specified as either `[user@]hostname` or a URI of the form `ssh://[user@]hostname[:port]`.
+    ///
+    /// The function provide interactive authentication based on `STDIN` (such as reading a
+    /// password).
+    ///
+    /// Warning ⚠️: current use sshpass non-interactive ssh password auth.
+    /// See detail: <https://sourceforge.net/projects/sshpass/>
+    ///
+    /// The current implementation of sshpass is only temporary, and the functionality will be
+    /// optimized later. If using this feature, pay attention to security risks.
+    ///
+    /// For more options, see [`SessionBuilder`].
+    #[cfg(feature = "process-mux")]
```

Comment:
I'd like to put it under another feature "ssh-pass-login" since in the future we will implement this ourselves using pty, which will pull in additional features most don't use.